common/reviews/api/rush-lib.api.md

```diff
@@ -34,11 +34,11 @@ export class ApprovedPackagesPolicy {
     //
     // @internal
     constructor(rushConfiguration: RushConfiguration, rushConfigurationJson: IRushConfigurationJson);
-    readonly browserApprovedPackages: ApprovedPackagesConfiguration;
-    readonly enabled: boolean;
-    readonly ignoredNpmScopes: Set<string>;
-    readonly nonbrowserApprovedPackages: ApprovedPackagesConfiguration;
-    readonly reviewCategories: Set<string>;
+    get browserApprovedPackages(): ApprovedPackagesConfiguration;
```

Comment:
@sachinjoseph  Definitely do NOT upgrade TypeScript for `rush-lib`. 

FYI upgrading from TypeScript 3.5 -> 3.7 introduces these getter/setters that make the library unusuable by any project that uses a compiler older than TypeScript 3.6. See [this thread](https://twitter.com/octogonz_/status/1217565789252800512) for details.

Such a change would need to be a more significant version bump for Rush.  

@iclanton FYI

Reply:
Good point. Completely reverted all RSC upgradation changes in this PR.